src/presentation/dtos/device-dto.ts

```diff
@@ -0,0 +1,65 @@
+import {INewsDevice, TelemetricsDevice} from '../../model/entities/device'
+import {DeviceType} from '../../model/enums/device-type'
+import {StatusCode} from '../../model/enums/status-code'
+
+export interface DeviceDtoInterface {
+  type: DeviceType
+  id: string
+  name: string
+  isConnected: boolean
+  statusCode: StatusCode
+  statusMessage: string
+}
```

Comment:
These DeviceDtoInterfaces (for INews and Telemetrics as well) seems redundant. We don't make interfaces for our Dtos.
 
You use them when in your constructors when you want to parse from a `Device` to a `DeviceDto`. Simply just parse your `Device` into your constructor (which is the exact your are doing the places you create a new `DeviceDto`)